plugins/source/github/resources/services/users/users.go

```diff
@@ -0,0 +1,35 @@
+package users
+
+import (
+	"context"
+
+	"github.com/cloudquery/cloudquery/plugins/source/github/client"
+	"github.com/cloudquery/plugin-sdk/v4/schema"
+	"github.com/cloudquery/plugin-sdk/v4/transformers"
+	"github.com/google/go-github/v59/github"
+)
+
+func Users() *schema.Table {
+	return &schema.Table{
+		Name:     "github_users",
+		Resolver: fetchUser,
+		// No multiplexer is needed here because we are fetching the current user
+		Transform: client.TransformWithStruct(&github.User{}, transformers.WithPrimaryKeys("ID")),
+		Columns:   []schema.Column{client.OrgColumn},
+		Relations: []*schema.Table{keys()},
+	}
+}
+
+func fetchUser(ctx context.Context, meta schema.ClientMeta, _ *schema.Resource, res chan<- any) error {
```

Comment:
Can we move this code to the discovery/client init phase? That way we don't need the extra table. Not sure it's that useful

Reply:
My thought is if we have to make the API call then might as well persist the data

Reply:
I think this would also serve as a good connection test when someone is using token based auth